runtime/compiler/optimizer/abstractinterpreter/AbsInterpreter.cpp

```diff
@@ -0,0 +1,3072 @@
+                                                      comp(),
+                                                      depth,
+                                                      allConsts);
+         break;
+      case TR::MethodSymbol::Kinds::Interface:
+         {
+         callSite = new (region()) TR_J9InterfaceCallSite(_callerMethod,
+                                                            callNodeTreeTop,
+                                                            parent,
+                                                            callNode,
+                                                            interfaceMethod,
+                                                            calleeClass,
+                                                            vftSlotIndex,
+                                                            cpIndex,
+                                                            resolvedCalleeMethod,
+                                                            calleeSymbol,
+                                                            isIndirect,
+                                                            isInterface,
+                                                            info,
+                                                            comp(),
+                                                            depth,
+                                                            allConsts);
+         callSite->_isInterface = true;
+         break;
+         }
+      }
+
+   if (!callSite)
+      return NULL;
+
+   callSite->_byteCodeIndex = bcIndex;
+   callSite->_bcInfo = info;
+   callSite->_cpIndex= cpIndex;
+
+   return callSite;
+   }
+
+
+TR::SymbolReference* TR::AbsBlockInterpreter::getSymbolReference(int32_t cpIndex, TR::MethodSymbol::Kinds kind)
+   {
+   TR::SymbolReference *symbolReference = NULL;
+   switch(kind)
+      {
+      case TR::MethodSymbol::Kinds::Virtual:
+         symbolReference = comp()->getSymRefTab()->findOrCreateVirtualMethodSymbol(_callerMethodSymbol, cpIndex);
+         break;
+      case TR::MethodSymbol::Kinds::Static:
+         symbolReference = comp()->getSymRefTab()->findOrCreateStaticMethodSymbol(_callerMethodSymbol, cpIndex);
+         break;
+      case TR::MethodSymbol::Kinds::Interface:
+         symbolReference = comp()->getSymRefTab()->findOrCreateInterfaceMethodSymbol(_callerMethodSymbol, cpIndex);
+         break;
+      case TR::MethodSymbol::Kinds::Special:
+         symbolReference = comp()->getSymRefTab()->findOrCreateSpecialMethodSymbol(_callerMethodSymbol, cpIndex);
+         break;
+      default:
+         TR_ASSERT_FATAL(false, "Unexpected method symbol kind");
+         break;
+
+      }
+   return symbolReference;
+   }
+
+TR::AbsValue* TR::AbsBlockInterpreter::createObject(TR_OpaqueClassBlock* opaqueClass, TR_YesNoMaybe isNonNull)
+   {
+   TR::VPClassPresence *classPresence = isNonNull == TR_yes ? TR::VPNonNullObject::create(vp()) : NULL;
```

Comment:
I think the former contributor just thought that `TR_maybe` can better represent that we don't know if the object is null. He may also would like to align the method signature with `createArrayObject()` and `createStringObject()` since they also have a `isNonNull` parameter.

I have amended the code as your recommendation by using boolean to indicate `isNonNull` and adding a new method `createNonNullObject()`.